pages/_language/events/_event.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <main class="event">
+    <h2 class="event__title">{{ event.title }}</h2>
+    <nav>
+      <ul>
+        <li v-for="{ locale } in event._allTitleLocales" :key="locale">
+          <a
+            rel="alternate"
+            :hreflang="locale"
+            :lang="locale"
+            :href="`/${locale}/events/${params.event}`"
+          >
+            {{ locale }}
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </main>
+</template>
+
+<script>
+  import fetchContent from '~/lib/fetch-content';
+
+  export default {
+    layout: 'empty',
```

Comment:
Maybe already name this layout to 'event' or something similar?

Reply:
https://github.com/Deltares/wwa/pull/145/commits/0a3bc9c648f2383f770987d860fabbe485b25db7